launch/docker_test.go

```diff
@@ -303,5 +495,25 @@ func TestHelperProcess(t *testing.T) {
 			os.Exit(1)
 		}
 		os.Exit(0)
+	case "SUCCESS_TO_KILL":
+		if subcmd == "command" {
+			time.Sleep(fakeProcessLifeTime)
+			os.Exit(0)
+		}
+		os.Exit(0)
+	case "SUCCESS_TO_KILL_DOCKER_CMD":
+		if subcmd == "sleep" {
+			time.Sleep(fakeProcessLifeTime)
+		}
+	case "FAILED_TO_KILL":
```

Comment:
It would be better to combine it with the others.
```suggestion
	case "FAIL_TO_KILL":
```